internal/runner/client/client_local.go

```diff
@@ -218,6 +171,43 @@ func shellID() (int, bool) {
 	return i, true
 }
 
+func (r *LocalRunner) setupAdHocServer() (
```

Comment:
We want to extend the existing [Runner interface](https://github.com/stateful/runme/blob/main/internal/runner/client/client.go#L55-L66) that separates Local and Remote implementations, not make the Local implementation Remote.